test/runtests.jl

```diff
@@ -1,6 +1,40 @@
 using TZJData
+using TimeZones: TZJFile, TimeZone, Class
 using Test
 
+# TODO: Can be removed once this is in an official TimeZones release
```

Comment:
The release this will be included in is TimeZones.jl 1.12.0. Unfortunately, that release is blocked on a 1.0 release of this package which I'd like to have include these tests. We'll merge these tests as is.